benchmark/v4.0-performance-benchmarking-with-tpcc.md

```diff
@@ -14,7 +14,7 @@ draft: true
 
```

Comment:
The `draft: true` in meta info need to be deleted if. you want this file public on website. @JaySon-Huang

Reply:
I have removed that for TPC-H benchmark result. The TPC-C one should be confirmed by @innerr @Connor1996 .

Reply:
And not adding TPC-H result to `TOC.md` until @shuke987 @zz-jason confirm that we can show TPC-H result on our website.

Reply:
Got it.